build.gradle

```diff
@@ -161,4 +161,6 @@ dependencies {
     androidTestCompile "com.android.support.test:rules:0.3"
     androidTestCompile "com.android.support.test:runner:0.3"
     androidTestCompile "com.squareup.assertj:assertj-android:1.1.0"
+
+    compile 'com.google.zxing:android-integration:3.2.1'
```

Comment:
Please follow the common formatting style. Dependencies are listed alphabetically, grouped by Gradle specifier. And double quotes of course.